apps/meteor/client/views/marketplace/MarketplaceRouter.tsx

```diff
@@ -7,13 +7,20 @@ import PageSkeleton from '../../components/PageSkeleton';
 
 const MarketplaceRouter = ({ children }: { children?: ReactNode }): ReactElement => {
 	const currentContext = useRouteParameter('context');
-	const marketplaceRoute = useRoute('marketplace-all');
+	const marketplaceRoute = useRoute('marketplace-explore');
+	const isAppsEngineEnabled = useMethod('apps/is-enabled');
 
 	useEffect(() => {
-		if (!currentContext) {
-			marketplaceRoute.replace({ context: 'all', page: 'list' });
-		}
-	}, [currentContext, marketplaceRoute]);
+		const initialize = async () => {
+			// The currentContext === 'all' verification is for users who bookmarked
+			// the old marketplace
+			if ((!currentContext && (await isAppsEngineEnabled())) || currentContext === 'all') {
```

Comment:
Why is the verification of `isAppsEngineEnabled` needed for one condition (`!currentContext`) and not needed for the second one (`currentContext === 'all'`) ?

Also, is it really necessary to include those users who bookmarked? This pendency will be here forever..?

Reply:
First part: Oversight on my side;
Second part: I believe it is necessary since marketplace doesn´t really have fallbacks for wrong routes as of yet and I don´t think is a nice experience for our users to browse blank pages even if they are easy to get out of, wdyt?

Reply:
Maybe add a TO DO comment, to remove this once the fallback is implemented?

Reply:
Absolutely, Ill add this and then solve these changes, thanks for the input Julia.